mslib/mswms/demodata.py

```diff
@@ -942,6 +942,7 @@ def create_server_config(self, detailed_information=False):
 import mslib.mswms.dataaccess
 from mslib.mswms import mpl_hsec_styles
 from mslib.mswms import mpl_vsec_styles
+from mslib.mswms import mpl_1sec_styles
```

Comment:
I would prefer a consistent naming across the modules. I saw O OneD and 1 here. Also, the 1-D plot is no "cross-section" anymore...? Maybe simply mpl_oned_styles?